tlslite/utils/rsakey.py

```diff
@@ -87,6 +91,167 @@ def hashAndVerify(self, sigBytes, bytes):
         result2 = self.verify(sigBytes, prefixedHashBytes2)
         return (result1 or result2)
 
+    def MGF1(self, mgfSeed, maskLen, hAlg):
+        """Generate mask from passed-in seed.
+
+        This generates mask based on passed-in seed and output maskLen.
```

Comment:
Information where the definition of the method comes from, where the algorithm is defined.